src/scores/probability/crps_impl.py

```diff
@@ -1094,3 +1096,101 @@ def _vfunc(x, threshold=threshold):
         weights=weights,
     )
     return result
+
+
+def interval_tw_crps_for_ensemble(
+    fcst: XarrayLike,
+    obs: XarrayLike,
+    ensemble_member_dim: str,
+    lower_threshold: Union[xr.DataArray, float],
+    upper_threshold: Union[xr.DataArray, float],
+    *,  # Force keywords arguments to be keyword-only
+    method: Literal["ecdf", "fair"] = "ecdf",
+    reduce_dims: Optional[Sequence[str]] = None,
+    preserve_dims: Optional[Sequence[str]] = None,
+    weights: Optional[XarrayLike] = None,
+) -> XarrayLike:
+    """
+    Calculates the threshold weighted continuous ranked probability score (twCRPS)
+    weighted for an interval across the distribution from ensemble input.
+
+    A threshold weight of 1 is assigned for values within the interval and a threshold weight of 0 otherwise.
+    The threshold values that define the bounds of the interval are given by the
+    ``lower_threshold`` and ``upper_threshold`` arguments.
+    For example, if we only want to foucs on the temperatures between -10 and -20 degrees C
+    where aircraft icing is most likely, we can set ``lower_threshold=-20`` and ``upper_threshold=-10``.
+
+
+    For more flexible weighting options and the relevant equations, see the
+    :py:func:`scores.probability.tw_crps_for_ensemble` function.
+
+    Args:
+        fcst: Forecast data. Must have a dimension `ensemble_member_dim`.
```

Comment:
```suggestion
        fcst: Forecast data. Must have a dimension ``ensemble_member_dim``.
```
Is this meant to be two backticks?

Reply:
Thanks - I've also fixed this in other functions in the file.